src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/induction/InPrisonInterestsResourceMapper.kt

```diff
@@ -1,11 +1,26 @@
 package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.resource.mapper.induction
 
 import org.mapstruct.Mapper
+import org.mapstruct.Mapping
 import org.mapstruct.NullValueMappingStrategy
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.resource.mapper.InstantMapper
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.InPrisonInterests
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.dto.CreateInPrisonInterestsDto
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.CreateInPrisonInterestsRequest
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.InPrisonInterestsResponse
 
-@Mapper(nullValueIterableMappingStrategy = NullValueMappingStrategy.RETURN_DEFAULT)
+@Mapper(
+  uses = [
+    InstantMapper::class,
+  ],
+  nullValueIterableMappingStrategy = NullValueMappingStrategy.RETURN_DEFAULT,
+)
 interface InPrisonInterestsResourceMapper {
   fun toCreateInPrisonInterestsDto(request: CreateInPrisonInterestsRequest, prisonId: String): CreateInPrisonInterestsDto
+
+  @Mapping(target = "updatedBy", source = "lastUpdatedBy")
+  @Mapping(target = "updatedByDisplayName", source = "lastUpdatedByDisplayName")
+  @Mapping(target = "updatedAt", source = "lastUpdatedAt")
+  @Mapping(target = "updatedAtPrison", source = "lastUpdatedAtPrison")
+  fun toInPrisonInterestsResponse(inPrisonInterests: InPrisonInterests?): InPrisonInterestsResponse?
```

Comment:
The domain to model mapping methods in this PR are tiny (since they closely match), meaning we do not need any logic in abstract classes. Only the updated/lastUpdated fields are different. I've added corresponding unit tests mainly for regression purposes.